src/test/java/ru/all_easy/push/telegram/api/controller/UpdateControllerTest.java

```diff
@@ -0,0 +1,213 @@
+package ru.all_easy.push.telegram.api.controller;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.math.BigDecimal;
+import java.util.*;
+import org.junit.jupiter.api.*;
+import org.mockito.ArgumentCaptor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.Import;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.utility.DockerImageName;
+import redis.clients.jedis.Jedis;
+import ru.all_easy.push.common.client.model.SendMessageInfo;
+import ru.all_easy.push.expense.repository.ExpenseRepository;
+import ru.all_easy.push.room.repository.RoomRepository;
+import ru.all_easy.push.telegram.api.controller.model.*;
+import ru.all_easy.push.telegram.api.service.TelegramService;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@AutoConfigureMockMvc
+@Import(TestConfig.class)
```

Comment:
lets move all this annotations to some abstract class to not write them each time